src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -22,7 +22,8 @@
 <template>
 	<!-- Message Actions -->
 	<div v-click-outside="handleClickOutside"
-		class="message-buttons-bar">
+		class="message-buttons-bar"
+		:class="{ 'message-buttons-bar--last' : isLastRead }">
```

Comment:
```suggestion
		:class="{ 'message-buttons-bar--last-read' : isLastRead }">
```

Just to avoid confusion with it being the last message, CSS block also needs adjusting